src/lib/coda_intf/transition_handling_intf.ml

```diff
@@ -0,0 +1,402 @@
+open Core
```

Comment:
Can this file be named something else, like Transition_frontier_components?

Reply:
done